lib/ReactViews/Analytics/PolygonParameterEditor.jsx

```diff
@@ -79,7 +79,7 @@ const PolygonParameterEditor = React.createClass({
             points.push(CesiumMath.toDegrees(cartographic.latitude));
             pointsLongLats.push(points);
         }
-        this.props.parameterValues[this.props.parameter.id] = pointsLongLats;
+        this.props.parameterValues[this.props.parameter.id] = [pointsLongLats];
```

Comment:
Yes, that sounds good. And here we are changing it in an object in response to a click, not in render, so it should be ok.
It still feels a little mucky to me... I think I'd prefer to have a function you can call on the model itself to update its parameter values. But admittedly there's not much difference between setting it directly and calling a function to set it.
I'm happy not to raise this as an issue.